Version5 support (#111)
rzo1/crawler4j: README.md

## Table of content

> - [Why use this fork?](#why-use-this-fork)

Reviewer: ```suggestion
- [Why you should use this fork?](#why-you-should-use-this-fork)
```

- [Installation](#installation)
- [Quickstart](#quickstart)   
- [More Examples](#more-examples)
- [Configuration Details](#configuration-details)
- [Reconstructing extra urls to crawl](#reconstructing-extra-urls-to-crawl)
- [Authentication](#authentication)
- [High-level design diagrams](#high-level-design-diagrams)
- [License](#license)

> ## Why use this fork?

Reviewer: ```suggestion
## Why you should use this fork?
```

> This fork starts where development on the previous main repository stalled.

Reviewer: ```suggestion
This fork starts where the development of the previous main repository stalled.
```

> Some highlights:

Reviewer: ```suggestion
Some highlights include:
```

> - choice of frontiers => avoid using a database with a license that doesn't comply with your use-case

Reviewer: ```suggestion
- choice between multiple frontier implementation => avoid using a database with a license that doesn't comply with your use-case
```

> - easily substituting various parser implementations (not only for html, but also css, binary, and plain text)

Reviewer: ```suggestion
- easy substitution of various parser implementations (not only for html, but also css, binary, and plain text)
```

- dynamic authentication
- improved exception handling, more versatile to customize
- fixes various parsing issues
- more documentation
- more tests (JUnit)

> - uses Maven

Reviewer: ```suggestion
- uses Apache Maven as build tool
```

> - provides clean upgrade paths by keeping backward compatibility in mind and deprecating methods before removing them

Reviewer: ```suggestion
- provides a clean upgrade paths by keeping backward compatibility in mind and deprecating methods before removing them
```

- more eyes have gone through the code, so readability and correctness have improved

> - maintained, so dependencies are often updated to their latest versions

Reviewer: ```suggestion
- maintained, i.e. dependencies are often updated to their latest versions
```
